cfc-testing/truth-table-generator.py

```
@@ -0,0 +1,687 @@
+        # Generate truth table
+        self.truth_table = self.generate_truth_table()
+        
+        # Save outputs
+        json_file = self.save_truth_table("truth-table.json")
+        html_file = self.generate_html_report("truth-table-report.html")
+        
+        # Print summary
+        valid_count = len([c for c in self.truth_table.values() if c["valid"]])
+        invalid_count = len([c for c in self.truth_table.values() if not c["valid"]])
+        
+        print(f"\n📊 Summary:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)